### DISCUSSION TITLE
60

### DISCUSSION THREAD

**Discussion by giscus** [Shane-Park/markdownBlog#183] [Comments]
# 60

Intro Mac의 기본 터미널이 원래 bash 였는데 zsh 로 바꼈다고 합니다. 저는 Mac을 사용한지가 얼마 안되서 처음부터 zsh를 사용 했는데요. 덕분에 bash 기준으로 인터넷에 널리 퍼져있는 많은 명령어들이 한번에 입력되지 않아 처음에 꽤나 고생을 했습니다. zsh를 더욱 편하게 사용해주는 oh-my-zsh 가 있다고 해서 이번에 설치 해 보았습니다. git을 terminal 에서 활용해보려고 하는데, 유투브 강의에 나오는 환경들이 뭔가 달라서 보니 oh-my-zsh 셋팅이 된 환경이었습니다. 설치하기전에 ~/.zshrc 에 있는 텍스트 내용들을 백업해두시길 추천합니다. oh-my-zsh가 설치되면서 싹 밀려버리는 듯 합니다. vi ~/.zshrc 저는 아래 2줄만 있어서 복사해두었습니다. ..

https://shanepark.tistory.com/60

**Comment by dev-child** [comment]
안녕하세요. 이대로 해서 적용은 아주 잘 되었습니다~ 감사합니다.
혹시 근데 파란색 부분 색상은 바꿀수 있을까요?

**Reply by ShanePark** [reply]
안녕하세요. iTerm 쓰고 계신다면, `Preferences -> Profiles -> Colors` 에서 변경이 가능해요.
`Colors` 에서 우측 하단 보시면 `Color Presets` 가 있는데, 그걸로 이것 저것 바꿔보시면서 마음에 드는거 찾으시는게 제일 편해요.
저는 [Dracula](https://raw.githubusercontent.com/mbadolato/iTerm2-Color-Schemes/master/schemes/Dracula.itermcolors) 사용하고 있어요 :)
